CompuCell3D/core/Demos/experimental/CompuCellPythonTutorial/vectorPlot/error.txt

```diff
@@ -0,0 +1,12 @@
+Traceback (most recent call last):
```

Comment:
this error is caused by the fact that you are using SteppablePy as a base class. Let's fix if